c7n/resources/emr.py

```diff
@@ -75,6 +75,23 @@ def augment(self, resources):
 
 @actions.register('terminate')
 class Terminate(BaseAction):
+    """Action to terminate EMR cluster(s)
+
+    It is recommended to apply a filter to the terminate action to avoid
+    termination of all EMR clusters
+
+    :example:
+
+        .. code-block: yaml
+
+            policies:
+              - name: emr-terminate
+                resource: emr
```

Comment:
most of the prs are not commented, leaving a dozen prs in wip is not cool, especially while opening more wip prs. i'm closing out the ones that are not obviously correct at this point.